src/coreclr/vm/nativelibrarynative.cpp

```diff
@@ -1,71 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-//
-// File: NativeLibraryNative.cpp
-//
-
-#include "common.h"
-#include "nativelibrary.h"
-#include "nativelibrarynative.h"
-
-// static
-extern "C" INT_PTR QCALLTYPE NativeLibrary_LoadFromPath(LPCWSTR path, BOOL throwOnError)
```

Comment:
Would it be better to split this into unifying NativeLibrary_LoadFromPath/NativeLibrary_FreeLib/NativeLibrary_GetSymbol in one straightforward PR, and do the policy-heave APIs in a separate PR that is unlikely to be straightforward and that will need very careful review?

Reply:
Yeah I think so. The detailed behavior of LoadBySearch are not exactly the same. If we feel too complex we can also keep it unmanaged for coreclr.